### PR TITLE
Yolo model and objectDetection abstract class add

### PR DESCRIPTION
**Pull request by luisdavidparra**

### CODE REVIEW

File: MACHINE-LEARNING-SERVICE/src/helpers/filterResults.helper.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+2643 Av Melchor Perez de Olguin Colquiri Sud, Cochabamba, Bolivia.
+Av. General Inofuentes esquina Calle 20,Edificio Union № 1376, La Paz, Bolivia
+All rights reserved
+This software is the confidential and proprietary information of
+Jalasoft Confidential Information You shall not
+disclose such Confidential Information and shall use it only in
+accordance with the terms of the license agreement you entered into
+with Jalasoft
+*/
+
```
Comment (florpadilla) [inline MACHINE-LEARNING-SERVICE/src/helpers/filterResults.helper.js:12]: This class has not comments yet

**Review by florpadilla — COMMENTED** [review]
It looks good. Just please add comments in the "filterResults.helper"

File: MACHINE-LEARNING-SERVICE/src/helpers/filterResults.helper.js (hunk 1 above)
Comment (luisdavidparra) [reply MACHINE-LEARNING-SERVICE/src/helpers/filterResults.helper.js:12]: Done

Comment (florpadilla) [reply MACHINE-LEARNING-SERVICE/src/helpers/filterResults.helper.js:12]: Great!

**Pull request merged**